src/Admin/Views/Providers/Organizations.cshtml

```diff
@@ -37,7 +36,7 @@
                         {
                             <tr>
                                 <td class="align-middle">
-                                    <a asp-controller="Organizations" asp-action="Edit" asp-route-id="@org.OrganizationId">@WebUtility.HtmlDecode(org.OrganizationName)</a>
+                                    <a asp-controller="Organizations" asp-action="Edit" asp-route-id="@org.OrganizationId">@org.DecodedOrganizationName()</a>
```

Comment:
💭  (non-blocking) This could also follow the `DisplayName` naming scheme, rather than `Decoded` which is a bit less clear. `OrganizationDisplayName`.

Reply:
I forgot about that method, thanks for pointing it out. I have updated it.